content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,247 @@
+
+These analytics show the exact usage of your Sourcegraph instance so you can see which features are most used across your organization. The usage data is also used to quantify how much time developers have saved as they use Sourcegraph to search and understand code, run batch changes, create insights, and more. You can also customize the default time-saved values to reflect how your team uses Sourcegraph. 
+
+With in-product analytics, you can now report on:
+
+With charts that detail usage over time and value calculators that quantify the amount of time saved as users search, create insights, run batch changes, and more, you can now report on:
+- The number of searches developers have run in Sourcegraph, and how much development time has been saved as a result.
+- How many Code Insights charts have been created, viewed, and interacted with.
+- The number of changesets that have been created and merged via Batch Changes, and how much time has been saved as a result.
+- How frequently code navigation has been used by developers to quickly understand code.
+- How often IDE and browser extensions are used across your organization, and how much time has been saved as a result.
+- The number of navigation events and the percentage of your repositories that allow precise code navigation.
+
+<div className="tw-mb-sm">
+  <Figure
+    src="https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Search.png"
+    alt="In-product analytics"
+    caption="Understand usage and quantify the value of using Sourcegraph with in-product analytics"
+  />
+</div>
+
+<a href="https://docs.sourcegraph.com/admin/analytics" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<Badge link="https://docs.sourcegraph.com/admin/observability/opentelemetry" text="Admin" color="violet" size="small" />
+
+#### Easily export traces using OpenTelemetry
+
+[OpenTelemetry (OTEL)]((https://opentelemetry.io/)) is an open specification for exporting observability data. We’ve added OpenTelemetry support in Sourcegraph 4.0 so you can easily export tracing data from Sourcegraph into your preferred observability solution. This is a breaking change as we now require you to export tracing with the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/). This is deployed by default in all Sourcegraph deployment methods and replaces the Jaeger agents and collectors that was previously the default.
+
+<a href="https://docs.sourcegraph.com/admin/observability/opentelemetry" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<Badge link="https://docs.sourcegraph.com/admin/external_service" text="Admin" color="violet" size="small" />
+
+#### Quickly see the status on your repository and permissions syncing
```

Comment:
@ryphil @lcolston08 to be precise: this is unrelated to "permissions syncing", it's solely about repositories and syncing code from code hosts to Sourcegraph. I'd remove the "permissions syncing".

```suggestion
#### Quickly see the status on your repositories being synced
```